### ISSUE TITLE
[Rule] Kwarg-like arguments should be last

### ISSUE THREAD

**Issue by bhirsz** [MarketSquare/robotframework-robocop#474]
```
    [Arguments]  ${arg}  ${arg2}=3   # valid
```

```
    [Arguments]  ${arg}=3  ${arg2}   # invalid
```

**Comment by bhirsz** [comment]
Same with &{vars}

**Comment by bhirsz** [comment]
It is reported as:
```
0401 Robot Framework syntax error: Non-default argument after default arguments.
```
So it should be enough to filter our parsing-error and report it as another rule.

**Comment by bhirsz** [comment]
This:
```
Keyword3
    [Arguments]  &{kwargs}  @{args}
```
is reported as:
```
0401 Robot Framework syntax error: Only last argument can be kwargs. (parsing-error)
```

**Comment by mnojek** [comment]
> This:
> 
> ```
> Keyword3
>     [Arguments]  &{kwargs}  @{args}
> ```
> 
> is reported as:
> 
> ```
> 0401 Robot Framework syntax error: Only last argument can be kwargs. (parsing-error)
> ```

I would avoid using 'kwarg' in the message since I know that it is confusing for many people. Better to use 'positional arguments' for args and 'named arguments' for kwargs.

**Comment by bhirsz** [comment]
It is what Robocop currently reports through parsing-error (which is directly taken from Robot Framework parsing engine). I agree with you and we should create new rules and 'filter out' it from parsing-error with better messages.
